rest-new/src/main/java/org/jboss/pnc/rest/endpoints/BuildEndpointImpl.java

```diff
@@ -164,22 +168,32 @@ public void removeAttribute(int id, String key) {
 
     @Override
     public BuildPushResult getPushResult(int id) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+        return brewPusher.getBrewPushResult(id);
     }
 
     @Override
-    public Page<BuildPushResult> push(BuildPushRequest buildRecordPushRequest) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+    public Page<BuildPushResult> push(BuildPushRequest buildPushRequest) {
```

Comment:
I realized this should probably return only `BuildPushResult`, no? There are not multiple results when pushing a single build.

Reply:
Right! I'll change the API to return only one result.